src/one_dragon/base/screen/screen_loader.py

```
@@ -131,6 +134,57 @@ def reload(self, from_memory: bool = False, from_separated_files: bool = False)
+                log.debug(f"加载插件画面: {file_path}")
+                data = yaml_utils.safe_load(file)
+            if not isinstance(data, dict):
+                log.warning(f"插件画面配置格式错误，已跳过: {file_path}")
+                continue
+
+            if default_app_id and not data.get('app_id'):
+                data['app_id'] = default_app_id
+
+            screen_info = ScreenInfo(data)
+            if screen_info.screen_name in self.screen_info_map:
+                log.warning(f"插件画面名称冲突，已跳过: {screen_info.screen_name}")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)